nemesis/arena_prep.py

```diff
@@ -222,10 +223,19 @@ def battle_farm_all(ap_cp_limit: int = 255, yuna_attack=True, fayth_cave=True):
                         CurrentPlayer().defend()
                 else:
                     battle.main.escape_one()
-    memory.main.click_to_control()
-    if memory.main.get_hp()[0] < 1100:
-        battle.main.heal_up(3)
-    nemesis.menu.perform_next_grid(limit=ap_cp_limit)
+    if memory.main.game_over():
+        while memory.main.get_map() != 23:
+            xbox.tap_b()
+        new_game(gamestate="Nem_Farm")
+        load_game.load_save_num(0)
```

Comment:
I've been thinking about this. This will handle loading the save just fine, but will it handle things like rolling back the AP thresholds we need? Or any of the other game state stuff that we keep track of for nem runs?

Reply:
This is to load save number zero (not AP checkpoint zero). The AP checkpoint is updated in the parent function based on the return value of True or False in this block of code. (discussed further in Discord).